manifests/letsencrypt-issuer.yaml

```diff
@@ -0,0 +1,62 @@
+{{ if eq .dns.type "route53" }}
+---
+kind: Secret
+apiVersion: v1
+metadata:
+  name: prod-route53-credentials
+type: Opaque
+stringData:
+  secret-access-key: "{{.dns.secretKey}}"
+{{ end }}
+---
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+spec:
+  acme:
+    server: https://acme-v02.api.letsencrypt.org/directory
+    email: "{{.letsencrypt.email}}"
+    privateKeySecretRef:
+      name: letsencrypt-prod
+    solvers:
+      {{ if eq .dns.type "route53" }}
+      dns01:
+        route53:
+          region: "{{.dns.region}}"
+          hostedZoneID: "{{.dns.zone}}"
+          accessKeyID: "{{.dns.accessKey}}"
+          secretAccessKeySecretRef:
+            name: prod-route53-credentials
+            key: secret-access-key
+      {{ else }}
+      http01:
+        ingress:
+          class: nginx
```

Comment:
The nginx deployment isn't using ingress classes - it is just the default ingress